cmake/Config.cmake

```diff
@@ -19,8 +19,24 @@ if (WIN32)
             "${CMAKE_CURRENT_SOURCE_DIR}/cs/test/compat/bin/retail")
 endif()
 
-find_package (PythonLibs 2.7)
-find_package (PythonInterp 2.7)
+# find python interpreter and library
+# to specify a different version, invoke cmake with:
+# -DPython_ADDITIONAL_VERSIONS=Major.Minor
+find_package (PythonInterp ${Python_ADDITIONAL_VERSIONS})
```

Comment:
I tried to follow the cmake recommendation to not use any version at all, but did not work (at least on Ubuntu 14.04). Result matrix with the header corresponding to values in the `find_package(PythonInterp` **header value**`)`:

 | Blank | 2.7 | `${Python_ADDITIONAL_VERSIONS}` |
|---|---|---|---|---|
`cmake` | py27 | py27 | py27  |   |   |
`cmake -DPython_ADDITIONAL_VERSIONS=2.7` | py27 | py27 | py27 |   |   |
`cmake -DPython_ADDITIONAL_VERSIONS=3.4` | interp=py27, pylib=not found, boostpylib=py27 | interp=py27, pylib=not found, boostpylib=py27 | py34  |   |   |